token/cli/src/main.rs

```diff
@@ -5171,6 +5419,7 @@ mod tests {
     }
 
     #[tokio::test]
+    #[ignore]
```

Comment:
Did you mean to leave these `ignore`s in here?

Reply:
Oops! Removing them